examples/CMakeLists.txt

```diff
@@ -18,7 +18,7 @@ set(EXAMPLE_NAME umf_example_basic)
 add_umf_executable(
     NAME ${EXAMPLE_NAME}
     SRCS basic/basic.c
-    LIBS umf ${LIBHWLOC_LIBRARIES})
+    LIBS umf ${UMF_HWLOC_NAME})
```

Comment:
cant we just set LIBHWLOC_LIBRARIES = UMF_HWLOC_NAME ?

Reply:
No, as `LIBHWLOC_LIBRARIES` is set to a full path to the hwloc library and is used globally.

Reply:
right now, this could stay as it is, but I think we should add additional args to add_umf_executable() like LIB_DIRS and INCLUDE_DIRS and separate the name of the lib from the path